index.html

```diff
@@ -85,7 +85,21 @@ <h3 class="title-subheader">Aug 23-26  //  Portland, OR </h3>
 		</div>
 	</section>
 
-<!-- ### SITE NAV ### -->
+<!-- ## TABLET & DESKTOP NAV ### -->
+  <section class="nav-section">
+    <nav class="middleHeader">
```

Comment:
can we have the semantic html `nav` on a class that isn't removed? it would probably be good to do an audit of our semantic html once we have the base page finished...